src/backend-test/__tests__/index.ts

```diff
@@ -0,0 +1,20 @@
+import {synthSnapshot} from 'projen/lib/util/synth'
+import {OttofellerBackendTestProject, OttofellerBackendTestProjectOptions} from '..'
+
+describe('Backend-test template', () => {
+  test('sets defaults', () => {
+    const project = new TestBackendTestProject({hasGitHooks: true})
+    const snapshot = synthSnapshot(project)
+    expect(snapshot).toMatchSnapshot()
+  })
+})
```

Comment:
I'd add some more checks. At least for the custom options (isGraphqlEnabled, isAWSDynamoDBlEnabled) - to see if they work correctly and when a non-default value is provided they actually erase their functionality (dependencies/scripts/etc.).